tests/conftest.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import pytest
+import dataset
+from sqlalchemy import create_engine, MetaData
+from sqlalchemy.orm import sessionmaker
+import processors.base.config as config
+
+# Make fixtures available to all tests
+
+from tests.fixtures.files import file_fixture
+from tests.fixtures.trials import trial
+from tests.fixtures.sources import nct_source, fda_source
+from tests.fixtures.fda_approvals import fda_approval
+from tests.fixtures.fda_applications import fda_application
+from tests.fixtures.organizations import organization
+
+
+def teardown_database(engine):
+    metadata = MetaData(bind=engine)
+    metadata.reflect()
+    metadata.drop_all()
+
+
+def create_test_database(source_database_url, test_database_url):
+    engine = create_engine(source_database_url)
+    test_engine = create_engine(test_database_url)
+    metadata = MetaData()
+    metadata.reflect(engine)
+    metadata.create_all(test_engine)
+    return test_engine
```

Comment:
I haven't checked the current Travis build, just saying this because by loading the schema like this will require that, to run the tests, you need all OpenTrials' databases. This makes it more difficult not only for Travis, but also for people cloning our repositories and wanting to set it up. It would also make the tests slower, as it dumps the schema every time you run the tests.

Because of this, I think that loading the schema on demand (i.e. when the tests are run) creates more problems than they solve, so aren't worth it.

Please just create a script that, when run, will dump the schema and write them to some place (e.g. `tests/dbs`). The tests just need to clean the data in the DB, dumping and loading the schema from the other DBs should be separate tasks, not run on each test run.